admin/server/projects.go

```diff
@@ -261,13 +261,17 @@ func (s *Server) UpdateProject(ctx context.Context, req *adminv1.UpdateProjectRe
 	observability.AddRequestAttributes(ctx,
 		attribute.String("args.org", req.OrganizationName),
 		attribute.String("args.project", req.Name),
-		attribute.String("args.description", req.Description),
-		attribute.Bool("args.public", req.Public),
-		attribute.String("args.region", req.Region),
-		attribute.Int64("args.prod_slots", req.ProdSlots),
-		attribute.String("args.prod_branch", req.ProdBranch),
-		attribute.String("args.github_url", req.GithubUrl),
+		attribute.String("args.description", safeStr(req.Description)),
+		attribute.String("args.region", safeStr(req.Region)),
+		attribute.String("args.prod_branch", safeStr(req.ProdBranch)),
+		attribute.String("args.github_url", safeStr(req.GithubUrl)),
```

Comment:
If the fields are not changed, this will make the logs look like they're being set to `""`, which is confusing

Reply:
I assumed that since empty string is not possible it would be self-explanatory that fields weren't set but yeah agreed on the confusion. I will only update when field is present like boolean fields.